index.js

```diff
@@ -25,24 +25,20 @@ function NotesApplication(author) {
 this.create = function(note_content) 
```

Comment:
Your opening block should start on the same line as your method signature, same goes for your conditional statements and class definitions. 

create() { //notice the space between the closing parathensis and the opening brace?
}

NOT

create 
{
}

OR

if 
{
}